README.md

```diff
@@ -1,2 +1,87 @@
-# BookStore
-The Bookstore is a website similar to the "Awesome Books" website built in my Repositories. This is an MVP version of it that allows you to display a list of books,add and delete
+# Project Name
+
+BOOKSTORE
+
+# Description the project.
+
+This project is built with React Js.
+
+The Bookstore is a website similar to the "Awesome Books" website built bymyself in my Github repositories. This is an MVP version of it that allows users to display a lists of books, add a book and remove a selected book.
+
+## Built With
+
+- React Js
+- Jest testing
+- following github flows standards.
+
+## Getting Started
+
+**This web page is books site with my List planned or shores to do in the future**
+
+**To do List project works on mobile and desktop using @Media queries .**
+
+---
+
+To get a local copy up and running follow these simple example steps.
+follow>>>>>>>>>
+
+### Setup
+
+to use the project To do list please follow :
+
+> Open GitHub and go to my GitHub repository name: math-magician.to clone.
+> Click “Code” and copy the given URL.
+> click code and copy the URL.
+
+> click code and copy the URL.
+
+> Open “Git Bash” and change the current working directory to the location where you want the cloned directory.
+
+> Type git clone in the terminal, paste the URL you copied earlier, and press “enter” to create your local clone.
+
+> Syntax:
+
+$ git clone {repository URL}
+
+### Usage
+
+> open your directory math-magician/
+
+\_use vs-code or any IDE to view line code .
+
+---
+
+$ npm start to run the local server
+
+---
+
+## Click here for live version
+
+- live demo: [Netlify](https://clinquant-taiyaki-41e17a.netlify.app/)
+
```

Comment:
- [ ]  Some information like the Project link and steps to configure this project are from the Math magician project, please update them and use the information for this project, not the previous project to have professional project documentation.